scripts/mod_loader/modapi/mechedit.lua

```diff
@@ -0,0 +1,509 @@
+	if not Game then
+		return false
+	end
+	
+	return CUtils.IsMechEditInventoryUpHovered()
+end
+
+function mechedit.IsInventoryDownHovered()
+	if not Game then
+		return false
+	end
+	
+	return CUtils.IsMechEditInventoryDownHovered()
+end
+
+function mechedit.IsSkill1Hovered()
+	if not Game then
+		return false
+	end
+	
+	return CUtils.IsMechEditSkill1Hovered()
+end
+
+function mechedit.IsSkill2Hovered()
+	if not Game then
+		return false
+	end
+	
+	return CUtils.IsMechEditSkill2Hovered()
+end
+
+function mechedit.IsAbilityHovered()
+	if not Game then
+		return false
+	end
+	
+	return CUtils.IsMechEditAbilityHovered()
+end
+
+function mechedit.SetTestHighlightable(highlightable)
+	Tests.AssertSignature{
+		ret = "void",
+		func = "SetTestHighlightable",
+		params = { highlightable },
+		{ "boolean|bool" }
+	}
+	
+	if not Game then
+		return
+	end
+	
+	CUtils.SetMechEditTestHighlightable(highlightable)
+end
+
+function mechedit.SetClassHighlightable(highlightable)
+	Tests.AssertSignature{
+		ret = "void",
+		func = "SetClassHighlightable",
+		params = { highlightable },
+		{ "boolean|bool" }
+	}
+	
+	if not Game then
+		return
+	end
+	
+	CUtils.SetMechEditClassHighlightable(highlightable)
+end
+
+function mechedit.SetHealthIconHighlightable(highlightable)
+	Tests.AssertSignature{
+		ret = "void",
+		func = "SetHealthIconHighlightable",
+		params = { highlightable },
+		{ "boolean|bool" }
+	}
+	
+	if not Game then
+		return
+	end
+	
+	CUtils.SetMechEditHealthIconHighlightable(highlightable)
+end
+
+function mechedit.SetMoveHighlightable(highlightable)
+	Tests.AssertSignature{
+		ret = "void",
+		func = "SetMoveHighlightable",
+		params = { highlightable },
+		{ "boolean|bool" }
+	}
+	
+	if not Game then
+		return
+	end
+	
+	CUtils.SetMechEditMoveHighlightable(highlightable)
+end
+
+function mechedit.SetMoveIconHighlightable(highlightable)
+	Tests.AssertSignature{
+		ret = "void",
+		func = "SetMoveIconHighlightable",
+		params = { highlightable },
+		{ "boolean|bool" }
+	}
+	
+	if not Game then
+		return
+	end
+	
+	CUtils.SetMechEditMoveIconHighlightable(highlightable)
+end
+
+function mechedit.SetNameHighlightable(highlightable)
+	Tests.AssertSignature{
+		ret = "void",
+		func = "SetNameHighlightable",
+		params = { highlightable },
+		{ "boolean|bool" }
+	}
+	
+	if not Game then
+		return
+	end
+	
+	CUtils.SetMechEditNameHighlightable(highlightable)
+end
+
+function mechedit.SetPilotNameHighlightable(highlightable)
+	Tests.AssertSignature{
+		ret = "void",
+		func = "SetPilotNameHighlightable",
+		params = { highlightable },
+		{ "boolean|bool" }
+	}
+	
+	if not Game then
+		return
+	end
+	
+	CUtils.SetMechEditPilotNameHighlightable(highlightable)
+end
+
+function mechedit.SetInstallHighlightable(highlightable)
+	Tests.AssertSignature{
+		ret = "void",
+		func = "SetInstallHighlightable",
+		params = { highlightable },
+		{ "boolean|bool" }
+	}
+	
+	if not Game then
+		return
+	end
+	
+	CUtils.SetMechEditInstallHighlightable(highlightable)
+end
```

Comment:
❗ Attempting to call this causes an error; it appears `CUtils.SetMechEditInstallHighlightable` is missing?

Reply:
Fixed in commit `c305de7`.
Note that this is another case where the function must be called each frame the vanilla behavior should be overridden.